tests/misalignedTest.js

```diff
@@ -0,0 +1,32 @@
+const { expect } = require("chai");
+const { print_color_map } = require("../misaligned.js");
+function colorMapTest() {
+  const { colorMap, majorColors, minorColors } = print_color_map();
+  console.log(colorMap);
+  const colorMapLines = colorMap.split("\n").filter(Boolean);
+  const majorColorArray = [...majorColors];
+  const minorColorArray = [...minorColors];
+  const maxSizeOfColorPair = majorColorArray.length * minorColorArray.length;
+  expect(colorMapLines.length).equals(25); //fails
+  expect(colorMapLines.length).equals(maxSizeOfColorPair); //fails
+  colorMapLines.forEach((pair, zeroBasedPairNumber) => {
+    const parts = pair.split("|");
```

Comment:
There is no test to see that the separators are aligned. Try adding an assert / expect for that